apps/retail/store/index.ts

```
@@ -13,25 +13,39 @@ import favoriteReducer from './favorite-slice'
+  activeMenuItem: activeMenuItemReducer,
+  settingBox: settingBoxReducer,
+  favorite: favoriteReducer,
+  transactionId: responseDataReducer,
+  quoteResponse: responseDataReducer,
+  customerDetails: responseDataReducer,
+  initResponse: responseDataReducer,
+  geoLocationSearchPageUI: geoMapLocationSearchReducer
+})
+
+// Root reducer with state reset on logout
+const rootReducer = (state: any, action: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)